<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformOptionsWidget</class>
 <widget class="QWidget" name="PlatformOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="mTargetOptionsGroupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Target Options</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="1">
       <widget class="UtQtAngleValueEdit" name="mTargetPitchEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="UtQtAngleValueEdit" name="mTargetYawEdit"/>
      </item>
      <item row="8" column="1">
       <widget class="UtQtAngleValueEdit" name="mTargetRollEdit"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Roll</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="UtQtLengthValueEdit" name="mTargetAltitudeEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="UtQtSpeedValueEdit" name="mTargetSpeedEdit"/>
      </item>
      <item row="9" column="1">
       <widget class="UtQtAngleValueEdit" name="mTargetHeadingEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="mTargetHeadingCheckBox">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Heading</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="mSensorControlGroupBox">
     <property name="title">
      <string>Sensor Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="9" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Roll</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="mAutoTargetCueingCheckBox">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>Automatic Target Cueing</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="UtQtAngleValueEdit" name="mSensorRollEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="mSensorPlatformToDetectEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Mode Name</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="UtQtAngleValueEdit" name="mSensorPitchEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Platform To Detect</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="UtQtAngleValueEdit" name="mSensorYawEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="mSensorModeComboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Required PD</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="mSensorPDSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="2">
       <widget class="UtQtLengthValueEdit" name="mSensorAltitudeEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="mFixedSensorCheckBox">
        <property name="text">
         <string>Fixed Sensor Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="mJammerControlGroupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Jammer Control</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Jammer to Signal Reference</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mJammerNameEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="UtQtRatioValueEdit" name="mJammerToSignalEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Jamming Platform Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtSpeedValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtAngleValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtRatioValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mTargetAltitudeEdit</tabstop>
  <tabstop>mTargetSpeedEdit</tabstop>
  <tabstop>mTargetYawEdit</tabstop>
  <tabstop>mTargetPitchEdit</tabstop>
  <tabstop>mTargetRollEdit</tabstop>
  <tabstop>mTargetHeadingEdit</tabstop>
  <tabstop>mTargetHeadingCheckBox</tabstop>
  <tabstop>mAutoTargetCueingCheckBox</tabstop>
  <tabstop>mSensorModeComboBox</tabstop>
  <tabstop>mSensorPDSpinBox</tabstop>
  <tabstop>mSensorPlatformToDetectEdit</tabstop>
  <tabstop>mSensorYawEdit</tabstop>
  <tabstop>mSensorPitchEdit</tabstop>
  <tabstop>mSensorRollEdit</tabstop>
  <tabstop>mJammerToSignalEdit</tabstop>
  <tabstop>mJammerNameEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
